### PR TITLE
[FEATURE]mongo sink - raw

### PR DESCRIPTION
**Pull request by iariasleon**

### CODE REVIEW

File: test/acceptance/tools/cygnus.py [hunk 1]
```diff
@@ -66,6 +68,8 @@
 CKAN_SINK        = u'ckan'
```
Comment (XavierVal) [inline test/acceptance/tools/cygnus.py]: Why do you assign a constant value to a variable value instead of using it directly?

File: test/acceptance/tools/cygnus.py [hunk 2]
```diff
@@ -519,12 +553,12 @@ def create_table (self, resource_name, service_path, attributes_number, attribut
         if resource_name == WITH_MAX_LENGTH_ALLOWED:
```
Comment (XavierVal) [inline test/acceptance/tools/cygnus.py]: The set of variable that are not going to change makes harder to read the conditions :S

**Comment by XavierVal** [comment]
You can reuse the pdi Mongo and Mysql libs, isn't it?

File: test/acceptance/README.md [hunk 3]
```diff
@@ -92,10 +94,10 @@ This file initially will overwrite properties.json in each feature.
 
 ### matching_table.conf
 
-We recommend to create your configurations lines in a copy file into settings folders and it will be copied to cygnus in conf folder, remotely.
+We recommend to create your  configurations lines in a copy file into settings folders and it will be copied to cygnus in conf folder, remotely.
```
Comment (frbattid) [inline test/acceptance/README.md]: Why adding this white space?

File: test/acceptance/README.md [hunk 4]
```diff
@@ -92,10 +94,10 @@ This file initially will overwrite properties.json in each feature.
...
 In cygnus will be called `matching_table.conf` by default (see configuration.json). If your tests do not use matching table implicitly,
 probably with to copy template to matching_table.conf is enough and then use `"default"` as file name value in step.
-`""` value (empty) in file name in step does not do anything and also removes the last matching_table.conf file if exists.
+`""` value (empty) in file name in step does not do anything and also removes the last matching_table.conf  file if exists.
```
Comment (frbattid) [inline test/acceptance/README.md]: Why adding this white space?

File: test/acceptance/integration/notifications/ckan/ckan_column_http.feature [hunk 5]
```diff
@@ -43,8 +43,8 @@ Feature: Stored in ckan new notifications per column from context broker
     And Verify that the attribute value is stored in ckan
   Examples:
     | organization   | service_path | attributesQuantity | attribute_name | attribute_type | attribute_value   | metadata_value | content |
-    | cygnus_col_023 | myserv6      | 1                  | pressure       | celcius        | 46.3              | True           | json    |
-    | cygnus_col_023 | myserv6      | 1                  | pressure       | celcius        | 46.3              | True           | xml     |
+    | cygnus_col_021 | myserv6      | 1                  | pressure       | celcius        | 46.3              | True           | json    |
```
Comment (frbattid) [inline test/acceptance/integration/notifications/ckan/ckan_column_http.feature]: Is it simply a change on the organizacion name?

File: test/acceptance/integration/notifications/ckan/ckan_column_http_matching_table.feature [hunk 6]
```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
```
Comment (frbattid) [inline test/acceptance/integration/notifications/ckan/ckan_column_http_matching_table.feature]: Since an in progress PR will change all the occurences of "matching table" by "grouping rules", this must be changed as well in these acceptance tests.

File: test/acceptance/integration/notifications/ckan/ckan_column_http_matching_table.feature [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+    And create a new resource "<new_destination>" with "1" attributes called "temperature", attribute type "celcius", attribute data type "json" and metadata data type "json"
+    And update real values in resource "<resource>" and service path "<service_path>" to notification request
+    When receives a notification with attributes value "random", metadata value "False" and content "<content>"
+    Then I receive an "OK" http code
+    And changes new destination "<new_destination>" where to verify in dataset "<new_dataset>"
+    And Verify that the attribute value is stored in ckan
+    And Verify the metadatas are stored in ckan
+    And delete matching table file
+  Examples:
+    | service_path | resource        | new_destination | new_dataset     | content |
```
Comment (frbattid) [inline test/acceptance/integration/notifications/ckan/ckan_column_http_matching_table.feature]: new_dataset --> new_service_path as stated in PR https://github.com/telefonicaid/fiware-cygnus/pull/409

File: test/acceptance/integration/notifications/ckan/ckan_column_http_matching_table.feature (hunk 6 above)
Comment (frbattid) [inline test/acceptance/integration/notifications/ckan/ckan_column_http_matching_table.feature]: The PR is https://github.com/telefonicaid/fiware-cygnus/pull/409

File: test/acceptance/integration/notifications/mongo/mongo_raw.feature [hunk 8]
```diff
@@ -0,0 +1,197 @@
...
+# iot_support at tid.es
+#
+__author__ = 'Iván Arias León (ivan.ariasleon at telefonica dot com)'
+
+#
+#  Note: the "skip" tag is to skip the scenarios that still are not developed or failed
+#        -tg=-skip
+#
+
+Feature: get raw values with differents requests from cygnus
```
Comment (frbattid) [inline test/acceptance/integration/notifications/mongo/mongo_raw.feature]: from --> using

File: test/acceptance/integration/notifications/mongo/mongo_raw.feature [hunk 9]
```diff
@@ -0,0 +1,197 @@
...
+  As a cygnus user
+  I want to be able to get raw values with differents requests from cygnus
+  so that they become more functional and useful
+
+
+  @happy_path
+  Scenario: get raw values using cygnus
```
Comment (frbattid) [inline test/acceptance/integration/notifications/mongo/mongo_raw.feature]: "Get raw values persisted by Cygnus"... I mean, Cygnus is not used for accessing the data :)

Comment (frbattid) [inline test/acceptance/integration/notifications/mongo/mongo_raw.feature]: Oh, wait, looking at the test, I've realized you want to receive a notification and persist into MongoDB the data within it, right? In that case, I think the scenario is "received a notification at Cygnus and store it as raw data".

Comment (frbattid) [inline test/acceptance/integration/notifications/mongo/mongo_raw.feature]: This comment applies (with minor changes) to all the scenario within this feature, I guess.

File: test/acceptance/README.md (hunk 3 above)
Comment (iariasleon) [inline test/acceptance/README.md]: I don't know... :-) 
It`s a typo

File: test/acceptance/README.md (hunk 4 above)
Comment (iariasleon) [inline test/acceptance/README.md]: It`s a typo

File: test/acceptance/integration/notifications/ckan/ckan_column_http.feature (hunk 5 above)
Comment (iariasleon) [inline test/acceptance/integration/notifications/ckan/ckan_column_http.feature]: Yes, by simetric with another tests in the same scenario, nothing important.

File: test/acceptance/integration/notifications/ckan/ckan_column_http_matching_table.feature (hunk 6 above)
Comment (iariasleon) [inline test/acceptance/integration/notifications/ckan/ckan_column_http_matching_table.feature]: Yes, but it will be in another PR... :-)

File: test/acceptance/integration/notifications/ckan/ckan_column_http_matching_table.feature (hunk 7 above)
Comment (iariasleon) [inline test/acceptance/integration/notifications/ckan/ckan_column_http_matching_table.feature]: Yes, these changes relative of grouping rules will be in another PR

File: test/acceptance/tools/cygnus.py (hunk 1 above)
Comment (iariasleon) [inline test/acceptance/tools/cygnus.py]: because, the sink names are constant values, for me is better define in constants, but it is only my opinion. I know that this behavior involves jumps in the code, I plan to do refactoring in the future

File: test/acceptance/tools/cygnus.py (hunk 2 above)
Comment (iariasleon) [inline test/acceptance/tools/cygnus.py]: I plan to do refactoring in the future.

File: test/acceptance/README.md (hunk 3 above)
Comment (iariasleon) [inline test/acceptance/README.md]: fixed in 3002b4d

File: test/acceptance/integration/notifications/mongo/mongo_raw.feature (hunk 8 above)
Comment (iariasleon) [inline test/acceptance/integration/notifications/mongo/mongo_raw.feature]: fixed in 3002b4d

File: test/acceptance/integration/notifications/ckan/ckan_column_http.feature (hunk 5 above)
Comment (frbattid) [inline test/acceptance/integration/notifications/ckan/ckan_column_http.feature]: NTC

File: test/acceptance/integration/notifications/ckan/ckan_column_http_matching_table.feature (hunk 6 above)
Comment (frbattid) [inline test/acceptance/integration/notifications/ckan/ckan_column_http_matching_table.feature]: Yes, just something to remind :)

NTC

File: test/acceptance/integration/notifications/ckan/ckan_column_http_matching_table.feature (hunk 7 above)
Comment (frbattid) [inline test/acceptance/integration/notifications/ckan/ckan_column_http_matching_table.feature]: Fine for me, NTC

**Comment by frbattid** [comment]
LGTM from my side!

**Comment by mcarracedo** [comment]
LGTM
